client/signup/steps/domains/index.jsx

```diff
@@ -625,6 +622,14 @@ class DomainsStep extends Component {
 		const { headerText, isAllDomains, siteType, isReskinned, stepSectionName, translate } =
 			this.props;
 
+		if ( stepSectionName === 'use-your-domain' ) {
```

Comment:
We have multiple instances of `use-your-domain` in this file, would it be a good opportunity to declare a constant to avoid use of multiple literal strings?

Reply:
I'm a little bit hesitant to do that, just because there are similar assumptions in other files about the name of that section (e.g. [here](https://github.com/Automattic/wp-calypso/blob/4ccce1f/client/components/domains/use-your-domain-step/index.jsx#L96)). It would indeed be nice to define that slug in a single location, but unless we do a larger refactor to achieve a "single source of truth" across many files, I would be inclined to leave this as is.